packages/usa-banner/src/styles/_usa-banner.scss

```diff
@@ -81,6 +81,20 @@ $banner-icon-close: (
       margin: 0;
     }
   }
+
+  &.usa-accordion__content > .grid-row {
```

Comment:
Do we need to be this specific or can we just target `grid-row` inside of `.usa-banner`?

```diff
-  &.usa-accordion__content > .grid-row {
# Move styles up
+ .usa-banner {
+   .grid-row {
```

Reply:
Looks like there's only the one `.grid-row` class within `usa-banner` so this works! I'll push that up now